en/04_Changelogs/5.2.0.md

```diff
@@ -115,6 +115,7 @@ You can also now optionally implement a `canView()` method on your `BuildTask` i
 ### Other new features
 
 - A new [`SiteTree.hide_pagetypes`](api:SilverStripe\CMS\Model\SiteTree->hide_pagetypes) configuration property has been added. Unlike [`SiteTree.hide_ancestor`](api:SilverStripe\CMS\Model\SiteTree->hide_ancestor) (which has [now been deprecated](#api-silverstripe-cms)), this is an array. This allows you to define all page types that should be hidden in a single configuration property in your yaml configuration.
+- A [`LogicException`](https://www.php.net/manual/en/class.logicexception.php) is now thrown by [`FormSchema::getSchema()`](api:SilverStripe\Forms\Schema::getSchema()) if a react component was not found for a field type.
```

Comment:
This really should have its own section, and probably even have a warning block since this _does_ have the chance to break some peoples' forms which are currently failing silently. Something like:
```text
[warning]
If your project or a module you're using is currently trying to include a field which doesn't have a react component into a react-rendered form, it will have been silently failing. The form will have been rendering everything except for the field(s) which have no react component.

This will now fail by throwing an exception, which means your form won't render at all until you remove or replace the field(s).
[/warning]
```

Reply:
Updated